src/App.jsx

```diff
@@ -1,31 +1,56 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
-
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+function getPreparedMovies(movies, { query }) {
+  const trimmedTitle = query.trim().toLowerCase();
+  const trimmedDescription = query.trim().toLowerCase();
```

Comment:
```suggestion
  const formattedQuery = query.trim().toLowerCase();
```

Why you need two the same variables 😄

Reply:
omg, sorry
I wrote this before my first cup of coffee((